C-Plus-Plus/cp/Minimum_steps_to_collect_coin_stacks.cpp

```diff
@@ -0,0 +1,67 @@
+/*Problem Statement: “Given an integer arrays representing the height of each stack of coins
+ and the number of coin stacks, calculate the minimum number of straight lines that pass 
+ through all the coins (minimum number of steps to collect these coins).”*/
+
+// Hints: Divide and Conquer
+#include <bits/stdc++.h>
+using namespace std;
+
+// returns minimum of two numbers
+int min2(int a, int b) {
+  return a < b ? a : b;
+}
```

Comment:
Min is already available in C++ in built Function
Directly use **min** & remove all this